### ISSUE TITLE
Post-purchase: Put plan features above plans grid 

### ISSUE THREAD

**Issue by rickybanister** [Automattic/jetpack#10381]
To reproduce:

I purchased a professional plan
Saw the thank you page, it completed set up
I clicked 'back to my site' (the left button on the purple card)
I landed in wp-admin on the jetpack plans page looking at the pricing grid:

![image](https://user-images.githubusercontent.com/437258/47383181-16469200-d6d2-11e8-8747-131a1c899e34.png)

I see that there's a manage plan button so I have purchased the plan, but all the 'next steps' are below the grid. 

What I expected:

To see the next step content above the plan grid. Can we swap the order of these for folks that purchase a plan until we have something better on wpcom for this flow?

cc @dereksmart @jeherve

**Comment by keoshi** [comment]
Shouldn't we try to match this to Calypso and introduce a “My Plan” tab with all the plan features below? My guess is that people wouldn't even expect to find a list of features included in their plan in a tab called “Plans”.

**Comment by rickybanister** [comment]
Yes probably, but we can do the above suggestion in about two minutes and marginally improve the flow ASAP vs wait for that development to happen.

**Comment by keoshi** [comment]
I reckon we can do both without development having to intervene except for the code review, but the alternative is way easier, I agree.

**Comment by rickybanister** [comment]
If you think we can split them up then let's do it, we'll have to change the referrer link post-purchase to be My Plan though.
